DaS-PC-MPChan/DarkSoulsProcess.vb

```diff
@@ -328,17 +335,30 @@ Public Class DarkSoulsProcess
             {&H058A71, 170},
             {&H18BD40, 250},
             {&H18CB21, 275},
-            {&H18CAD6, 256}
+            {&H18CAD6, 255}
         }
 
 
         Dim memory As New AllocatedMemory(_targetProcessHandle, code.Length)
         Dim jmpInstruction() As Byte = {&HE9}
         For Each returnLocation In returnLocations
+            If returnLocation.Key = &H18BD40 Then
+                jmpInstruction = {&HE8}
+            Else
+                jmpInstruction = {&HE9}
+            End If
             Dim jmpOffset As Int32 = (dsBase + returnLocation.Key) - (memory.address + returnLocation.Value) - 5
             Dim instruction() As Byte = jmpInstruction.Concat(BitConverter.GetBytes(jmpOffset)).ToArray()
             Array.Copy(instruction, 0, code, returnLocation.Value, instruction.Length)
         Next
+        
+        For Each bufferLocation In bufferLocations
+            'Could make another buffer but might as well use the spare space in the existing one
+            Dim bufOffset As Int32 = (memory.address + code.Length) + 20
```

Comment:
Oops you're right. I thought it had been allocated a fix amount of bytes rather than `code.Length` due to a volume of unused bytes after the end of the namecrash patches. Must just be an odd coincidence.